src/transport_ofi.c

```diff
@@ -1500,6 +1463,60 @@ int shmem_transport_init(void)
 int shmem_transport_startup(void)
 {
     int ret;
+    int i;
+    int  num_on_node = 0;
+
+    if (shmem_internal_params.OFI_STX_AUTO) {
+
+        ret = shmem_node_util_startup();
+        if (ret != 0) return ret;
+
+        num_on_node = shmem_node_util_n_local_pes();
+        if (num_on_node <= 0) {
```

Comment:
This check appears to be redundant, since this is already checked in `shmem_node_util_startup`.